rollup.config.js

```diff
@@ -0,0 +1,45 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import uglify from 'rollup-plugin-uglify';
+
+var env = process.env.NODE_ENV
+var config = {
+  format: 'umd',
+  moduleName: 'onClickOutside',
+  exports: 'named',
+  external: ['react'],
+  plugins: [
+    nodeResolve({
+      jsnext: true
+    }),
+    babel({
+      exclude: 'node_modules/**'
+    }),
+    replace({
+      'process.env.NODE_ENV': JSON.stringify(env)
+    })
+  ]
+}
+
+if (env === 'production') {
+  config.plugins.push(
+    uglify({
```

Comment:
this is not necessary, and in fact counter-productive: as a library that is itself used in larger bundles, *those* bundles are responsible for the minification, and can do a far better job at that when this library is not pre-uglified with its own settings.